pom.xml

```diff
@@ -75,15 +75,23 @@
                 <version>1.0.20</version>
                 <executions>
                     <execution>
-                        <goals>
-                            <goal>backport7to6</goal>
-                        </goals>
-                        <configuration>
-                            <classified>true</classified>
-                        </configuration>
+                        <!--<goals>-->
+                            <!--<goal>backport7to6</goal>-->
+                        <!--</goals>-->
+                        <!--<configuration>-->
+                            <!--<classified>true</classified>-->
+                        <!--</configuration>-->
                     </execution>
```

Comment:
Why commenting XML? If it's not needed, remove it. If it's maybe needed, put it in a profile.